composer.json

```diff
@@ -27,6 +27,7 @@
         "symfony/class-loader": "~2.3",
         "symfony/dom-crawler": "~2.3",
         "symfony/css-selector": "~2.3",
+        "phpunit/phpunit": "4.5.*",
```

Comment:
I think this should be `~4.5|~5.0` instead so that PHPUnit 5 is used for tests on PHP 7.